src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import style from './Counter.module.css';
+
+type Props = {};
+
+interface State {
+  counter: number,
+}
+
+class Counter extends React.Component<Props, State> {
```

Comment:
If there isn't Props u can write this:
```suggestion
class Counter extends React.Component<{}, State> {
```